demo/src/test/java/amazon/TaskFourTestThree.java

```diff
@@ -0,0 +1,86 @@
+package amazon;
+
+import com.google.common.collect.Ordering;
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TaskFourTestThree {
+
+
+    private WebDriver driver;
+
+    @BeforeTest
+    public void setUp(){
+        System.setProperty("webdriver.chrome.driver", "src/test/resources/webdriver/chromedriver.exe");
+        this.driver = new ChromeDriver();
+        this.driver.manage().window().maximize();
+    }
+
+    @Test
+    public void testThree(){
+
+
+        driver.get("https://www.amazon.com/s?k=school+supplies&crid=3B2SMJ2SQ64N&pd_rd_r=002c7e59-b3e2-4870-9de7-a9c54fb9263c&pd_rd_w=9XU4m&pd_rd_wg=DkrTY&pf_rd_p=c0f33dcc-b6ee-417c-b438-e358c4a5e8dc&pf_rd_r=BSVWZBX1JYQGKW4WR8F4&sprefix=school+su%2Caps%2C175&ref=pd_gw_unk");
```

Comment:
It is better to make this line a bit shorter, just move a part of this line to the next line

Reply:
Same for others